packages/flyout/src/getCoordinates.js

```diff
@@ -0,0 +1,262 @@
+import { anchorPoints } from "./anchorPoints";
```

Comment:
I could see this helper actually broken up into multiple like

`packages/flyout/src/coordinates/container`
`packages/flyout/src/coordinates/pointer`

etc